src/parcsr_ls/par_amg_setup.c

```diff
@@ -363,7 +363,7 @@ hypre_BoomerAMGSetup( void               *amg_vdata,
    /* probably should disable stuff like smooth num levels at some point */
 
 
-   if (grid_relax_type[0] >= 20) /* block relaxation choosen */
+   if (grid_relax_type[0] >= 20 && grid_relax_type[0] != 88) /* block relaxation choosen */
```

Comment:
We run out of the numbers for relaxation that are less than to equal to 20. I picked 88 as the new full-fledged l1 SHGS. Maybe we should think about alternative ways for "rlx_type" other than number-based ways.